_pages/en_US/godmode9-usage.txt

```diff
@@ -122,18 +122,52 @@ Insert the game cartridge you intend to dump into your device
   + **NDS Game Cartridge:** Press (A) on `[TitleID].trim.nds` to select it, then select "Copy to 0:/gm9/out"
 1. Your installable `.cia` or non-installable `.nds` formatted file will be outputted to the `/gm9/out/` folder on your SD card
 
+## Dumping a Game Cartridges romfs/exefs
```

Comment:
romfs and exefs aren't exactly things that a normal user would understand without context. Maybe use "dumping content from a game cartridge" or something